### PR TITLE
WIP: Improve error message when loading an invalid class version #13347

### PR DESCRIPTION
**Pull request by bhonsleaditya1**
Added major/minor/max version to J9CfrError

### CODE REVIEW

**Comment by pshipton** [comment]
FYI https://github.com/eclipse-openj9/openj9/blob/master/CONTRIBUTING.md
Note you need to sign the Eclipse ECA http://www.eclipse.org/legal/ECA.php in order to contribute.

File: runtime/oti/j9nonbuilder.h [hunk 1]
```diff
@@ -335,6 +335,9 @@ typedef struct J9CfrError {
 	I_32 errorBsmIndex;
 	U_32 errorBsmArgsIndex;
 	U_32 errorCPType;
+	U_16 majorVersion;
+    U_16 minorVersion;
```
Comment (pshipton) [inline runtime/oti/j9nonbuilder.h:339]: Pls follow the same formatting convention as used in the file. i.e. tabs instead of spaces.

**Comment by pshipton** [comment]
Note we follow the OMR coding standards https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md

**Comment by bhonsleaditya1** [comment]
@pshipton Can you please delete this PR since it contains my work mail instead of personal

**Comment by pshipton** [comment]
I can't delete it, you'd have to contact github for that. All we can do it close it.

**Comment by bhonsleaditya1** [comment]
Hi @pshipton I notified to GitHub Team they want the admin to initiate delete request for this PR. Please let me know the admin of this repo.

**Comment by pshipton** [comment]
@bhonsleaditya1 the Eclipse Foundation is the admin of the repo.

**Comment by bhonsleaditya1** [comment]
Could I get the handle or email of Eclipse Foundation? So that I can forward GitHub mail

**Comment by pshipton** [comment]
The Foundation is https://www.eclipse.org/
I expect you'll need to create an issue here https://bugs.eclipse.org/bugs/

**Pull request closed without merging**